doc/getting_started/installation_win.rst

```diff
@@ -39,7 +39,9 @@ packages from their respective websites.
 .. note::
    There are multiple ``set`` statements in this tutorial. You can avoid
    typing them every time by placing them inside a ``.cmd`` file and
-   running that every time you open a Command Prompt.
+   running that every time you open a Command Prompt. You can also
+   place them in :file:`%userprofile%\zephyr.cmd` and they will be executed
```

Comment:
you'll need to use an extra backslash here ``:file:`%userprofile%\\zephyr.cmd` ``